integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/TransferDashFragment.kt

```diff
@@ -198,7 +198,7 @@ class TransferDashFragment : Fragment(R.layout.transfer_dash_fragment) {
             hideBanners()
         }
 
-        binding.authLimitBanner.warningLimitInfo.setOnClickListener {
+        binding.authLimitBanner.root.setOnClickListener {
```

Comment:
Just handle the click on the entire surface of the banner. Having user to hit a small button on the edge of it is not UX friendly.